starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,63 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <link rel="stylesheet" type="text/css" href="C:\Users\Artsiom\Labs\lab-css-recipes-clone\starter-code\apple-pie-starter-code/css/main.css">
```

Comment:
Use this instead 
  <link rel="stylesheet" type="text/css" href="css/main.css">
This is a relative link.  The one you used will only work in your computer.